osu.Game/Skinning/SkinManager.cs

```diff
@@ -244,11 +250,11 @@ private T lookupWithFallback<T>(Func<ISkin, T> lookupFunction)
             // TODO: we also want to return a DefaultLegacySkin here if the current *beatmap* is providing any skinned elements.
             // When attempting to address this, we may want to move the full DefaultLegacySkin fallback logic to within Player itself (to better allow
             // for beatmap skin visibility).
```

Comment:
Why is this not moved to the new (correct) location?

Reply:
Ah forgot that.